drape_frontend/traffic_generator.hpp

```diff
@@ -20,25 +25,47 @@
 namespace df
 {
 
-enum TrafficSpeedBucket
+struct TrafficSegmentID
 {
-  Normal = 3,
-  Slow = 2,
-  VerySlow = 1
+  MwmSet::MwmId m_mwmId;
```

Comment:
MwmIds are quite expensive (they have shared_ptrs). Moreover, the number of different MwmIds is limited. Is it possible to split this class? I.e to replace vector of TrafficSegmentID by a map from MwmId to a vector of RoadSegmentIds?